cloudflare/resource_cloudflare_load_balancer_pool.go

```diff
@@ -314,7 +383,19 @@ func flattenLoadBalancerOrigins(d *schema.ResourceData, origins []cloudflare.Loa
 
 		flattened = append(flattened, cfg)
 	}
-	return schema.NewSet(schema.HashResource(originsElem), flattened)
+	return schema.NewSet(HashByMapKey("load_shedding"), flattened)
```

Comment:
this is the problematic bit; unsure why we're looking to return this value here as it doesn't seem related to the block it is in 🤔.